java/INSTALL

```diff
@@ -103,16 +108,16 @@ general overview of OpenDDS.
      use paths with no spaces (true on Unix too but spaces are rarely seen).
      The top level of the OpenDDS distribution is a directory that contains
      the subdirectory named dds.
-
+     
   3. Configure and build OpenDDS 
 
      Build as in $DDS_ROOT/install, but add --java option
 
-     Unix =>
-       ./configure --java=<full path to the JDK>
-     Windows => 
-       configure --java=<full path to the JDK>
-
+     Unix => cd $DDS_ROOT
+             ./configure --java
+             make       
+     Windows => configure --java=<full path to the JDK>
```

Comment:
OK, I finally find documentation for the perl module “File::Which”. The problem is that the module is not installed by default on my Mac or CentOS7 as far as I tested. Not sure if it’s installed by default on Windows as part of ActivePerl. I don’t have fresh windows to test it.

> On Sep 13, 2016, at 11:38 AM, Adam Mitz <notifications@github.com> wrote:
> 
> In java/INSTALL <https://github.com/objectcomputing/OpenDDS/pull/302#discussion_r78596769>:
> 
> >    3. Configure and build OpenDDS 
> >  
> >       Build as in $DDS_ROOT/install, but add --java option
> >  
> > -     Unix =>
> > -       ./configure --java=<full path to the JDK>
> > -     Windows => 
> > -       configure --java=<full path to the JDK>
> > -
> > +     Unix => cd $DDS_ROOT
> > +             ./configure --java
> > +             make       
> > +     Windows => configure --java=<full path to the JDK>
> Just like other environment variables that appear in "configure --help", it would be up to the user to set. Find the location of javac on the path (on all platforms) using the which() function of the configure script.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/objectcomputing/OpenDDS/pull/302/files/15a960d61ac519cc453485d4640d2aa233ce0494#r78596769>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ADwEI0kLVK2LhY0rCwubGmSb4QVhKQFUks5qptF2gaJpZM4J73YU>.
>